_episodes/04-faceting-and-filtering.md

```diff
@@ -58,15 +58,15 @@ You can also `invert` the filter to show all records which do not match your sel
 ## Filters
 As well as using Facets to filter the data displayed in OpenRefine you can also apply 'Text Filters' which looks for a particular piece of text appearing in a column  based on a unique text string, like a 'find' feature. Text filters are applied by clicking the drop down menu at the top of the column you want to apply the filter to and choosing 'Text filter'.
 
-As with Facets, the Filter options appear in the left hand panel in OpenRefine. Simply type in the text you want to use in the Filter to display only rows which contain that text in the relevant column.
+As with Facets, the Filter options appear in the left hand panel in OpenRefine. As you type the text you want to find into the Filter's text box, OpenRefine works to display only rows that contain that text in the relevant column.
```

Comment:
I prefer the description of "the text you want to use in the Filter" rather than "the text you want to find". The function of the text filter is to filter the rows to text to those that fulfil certain conditions (containing a text string or regular expression) rather than "find this text"